### PR TITLE
Chore: Add a spiner during packages installation

### LINKED ISSUE #894: Improvements to package installation
We can improve the experience of installing packages via `sonarwhal`:

* [ ] Add a visual indicator that things are happening. Installing a configuration can take a while and I've wondered more than once if the process was stuck
* [ ] Fix typo in the success message `Packages inStalled successfully`

### PR DESCRIPTION
**Pull request by sarvaje**
Fix #894

## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonarwhal)
- [x] Followed the [commit message guidelines](https://sonarwhal.com/docs/contributor-guide/getting-started/pull-requests/#commit-messagess)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [x] Added/Updated related tests.

### CODE REVIEW

**Comment by molant** [comment]
Tests seem to be broken?

**Comment by sarvaje** [comment]
>Tests seem to be broken?

Yep, working on it!

**Comment by sarvaje** [comment]
@molant tests should be fixed now

File: packages/sonarwhal/src/lib/utils/npm.ts [hunk 1]
```diff
@@ -1,17 +1,44 @@
...
+        const npmInstall = spawn(command, [], { shell: true });
+        let errors: string = '';
+
+        npmInstall.stderr.setEncoding('utf8');
+        npmInstall.stderr.on('data', (data) => {
+            errors += data;
+        });
+
+        npmInstall.on('error', (err) => {
+            reject(err);
```
Comment (molant) [inline packages/sonarwhal/src/lib/utils/npm.ts]: Can we have a `return` here just for consistency with the `exit`?

**Comment by sarvaje** [comment]
@molant done!

**Pull request closed without merging**